opensrp-app/src/main/java/org/smartregister/service/UserService.java

```diff
@@ -295,7 +295,7 @@ public LoginResponse isValidRemoteLogin(String userName, String password) {
         requestURL = configuration.dristhiBaseURL() + OPENSRP_AUTH_USER_URL_PATH;
 
         LoginResponse loginResponse = httpAgent
-                .urlCanBeAccessWithGivenCredentials(requestURL, userName, password);
+                .urlCanBeAccessWithGivenCredentials(requestURL.replaceAll("\\s+", ""), userName, password);
```

Comment:
why are you replacing this?

Reply:
Because codacy rejected reassigning parameter variables

Reply:
Made changes